foundationdb/src/database.rs

```diff
@@ -218,6 +229,76 @@ impl Database {
         )
     }
 
+    /// Runs a transactional function against this Database with retry logic.
+    /// The associated closure will be called until a non-retryable FDBError
+    /// is thrown or commit(), returns success.
+    /// This call is blocking -- this method will not return until commit() has been called and returned success.
```

Comment:
well it's async, so not really blocking :)

Reply:
This is misleading indeed, changing